src/main/java/edu/greenblitz/robotName/commands/shooter/shootingState/GoToRobotTargetPosition.java

```diff
@@ -0,0 +1,17 @@
+package edu.greenblitz.robotName.commands.shooter.shootingState;
+
+import edu.greenblitz.robotName.commands.auto.MoveToPose;
+import edu.greenblitz.robotName.commands.swerve.RotateToAngle;
+import edu.greenblitz.robotName.shootingStateService.ShootingState;
+import edu.greenblitz.robotName.utils.GBCommand;
+
+public class GoToRobotTargetPosition extends GBCommand {
```

Comment:
maybe
```suggestion
public class MoveRobotToShootingPosition extends GBCommand {
```
or something else